string-manipulation-2/src/num-vowels.js

```diff
@@ -0,0 +1,17 @@
+/* exported numVowels */
+function numVowels(string) {
+  const lowerCaseString = string.toLowerCase();
+  let counter = 0;
+  for (let i = 0; i < string.length; i++) {
+    if (
+      lowerCaseString[i] === 'a' ||
+      lowerCaseString[i] === 'e' ||
+      lowerCaseString[i] === 'i' ||
+      lowerCaseString[i] === 'o' ||
+      lowerCaseString[i] === 'u'
+    ) {
+      counter += 1;
+    }
+  }
+  return counter;
+}
```

Comment:
Totally works this way! Just want to show you another way of accomplishing this. Since we need to check for so many different characters, this would be a perfect example of when we'd want to use a `switch`
```
function numVowels(string) {
  let vowelCount = 0;
  for (let i = 0; i < string.length; i++) {
    switch (string[i].toLowerCase()) {
      case 'a':
      case 'e':
      case 'i':
      case 'o':
      case 'u':
        vowelCount++;
    }
  }
  return vowelCount;
}
```